src/style.css

```diff
@@ -2,0 +1,24 @@
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+}
+
+.container {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
Don't use tag names for styling. Instead of `.stars--1 .stars__star`, use a class to define the styles for these elements. This will make your CSS more maintainable and your elements more reusable.